File: dto-mapping-benchmark/src/main/java/com/example/dtomappingbenchmark/v1/repository/UserRepository.java
package com.example.dtomappingbenchmark.v1.repository;

import com.example.dtomappingbenchmark.v1.entity.User;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository("userRepositoryV1")
public interface UserRepository extends JpaRepository <User, Long>{
}

File: object-mapping-benchmark-v4/src/main/kotlin/yu/selab/cj/objectmappingbenchmarkv4/user/repository/UserRepository.kt
package yu.selab.cj.objectmappingbenchmarkv4.user.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import yu.selab.cj.objectmappingbenchmarkv4.user.entity.User

@Repository
interface UserRepository : JpaRepository<User, Long> {
}